### PR TITLE
Split out display factory from controllers

### PR DESCRIPTION
**Pull request by alex-deaconu**
[stage-2]

### CODE REVIEW

**Comment by alex-deaconu** [comment]
@ezequielc PR to prepare for the Add Display Instruction Email & Modal card. 

Please review. Thanks!

File: web/scripts/app.js [hunk 1]
```diff
@@ -284,9 +286,9 @@ angular.module('risevision.apps', [
         }],
         controller: 'displayAdd',
         resolve: {
-          canAccessApps: ['canAccessApps',
-            function (canAccessApps) {
-              return canAccessApps();
+          displayInfo: ['canAccessApps', 'displayFactory',
```
Comment (ezequielc) [inline web/scripts/app.js]: @alex-deaconu Curious, why does it need the `displayInfo` here? I think it is not being used as you inject the displayFactory to the controller. Thanks

Comment (alex-deaconu) [inline web/scripts/app.js]: @ezequielc it's consistent with what we have for Schedules: https://github.com/Rise-Vision/rise-vision-apps/blob/master/web/scripts/app.js#L196 (and in some ways Editor). Basically this is a dummy variable that allows us to check `canAccessApps()` and initialize the `displayFactory.newDisplay`. Any suggestions for refactoring this? Thanks.

Comment (ezequielc) [inline web/scripts/app.js]: @alex-deaconu Ok, I thought that maybe `canAccessApps()` alone would work, but I am good with the way it is. Thanks

**Comment by ezequielc** [comment]
@alex-deaconu Looks good :+1: Thanks!

File: web/scripts/app.js (hunk 1 above)
Comment (alex-deaconu) [inline web/scripts/app.js]: @ezequielc that was how I first implemented it. However I noticed that when adding a new Display, the previous Display's data would appear. So we have to reset the factory on a new Display. Thanks.

**Pull request merged**